DOWNSTREAM_OWNERS

```diff
@@ -0,0 +1,9 @@
+
+approvers:
+- openshift-team-operator-runtime
+- openshift-team-operator-runtime-maintainers
+
+reviewers:
+- openshift-team-operator-runtime
+- openshift-team-operator-runtime-maintainers
+
```

Comment:
I might be wrong, but I don't think these team names match up to anything related to GitHub teams or anything in prow. I think you would either need to create a separate DOWNSTREAM_OWNERS_ALIASES file to define the members of these teams, or just forego the DOWNSTREAM_OWNERS_ALIASES file and put GH user handles directly in this file.

Reply:
https://github.com/orgs/openshift/teams/openshift-team-operator-runtime
https://github.com/orgs/openshift/teams/openshift-team-operator-runtime-maintainers
?

Reply:
Sorry, I was not clear in my comment. I'm not sure if Prow knows how to handle github team names.

Reply:
https://docs.prow.k8s.io/docs/components/plugins/approve/approvers/#overview

> However, GitHub Team names are not supported. We do not use them because there is no audit log for changes to the GitHub Teams. This way we have an audit log.